src/main/java/org/broadinstitute/hellbender/tools/exome/AllelicCountTableColumns.java

```diff
@@ -6,7 +6,8 @@
  * Created by davidben on 11/30/15.
  */
 public enum AllelicCountTableColumns {
-    CONTIG("CONTIG"), POSITION("POS"), REF_COUNT("REF_COUNT"), ALT_COUNT("ALT_COUNT");
+    CONTIG("CONTIG"), POSITION("POS"), REF_COUNT("REF_COUNT"), ALT_COUNT("ALT_COUNT"), REF_NUCLEOTIDE("REF_NUCLEOTIDE"),
```

Comment:
So the only difference between name and string is POSITION vs POS... is it really worth the hassle? I would coalesce those two into one (I prefer position given the length of the other column names) and just relay on the constant name() to map between object and string.